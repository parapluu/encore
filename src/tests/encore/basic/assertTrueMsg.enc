-- This file was automatically converted by encorec

class Main
  def main() : void
    assertTrue(true)
    assertTrue(false, "Int {} Bool {} String {}", 42, false, "Foo")
  end
end

