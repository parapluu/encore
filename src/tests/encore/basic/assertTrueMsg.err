Assertion failed at "assertTrueMsg.enc" (line 6, column 5):
Int 42 Bool false String Foo
