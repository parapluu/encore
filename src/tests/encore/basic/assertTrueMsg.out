Assertion failed at "assertTrueMsg.enc" (line 4, column 5):
Int 42 Bool false String Foo
